guides/common/modules/proc_running-installer.adoc

```diff
@@ -0,0 +1,17 @@
+[id="running-installer_{context}"]
+= Running the Installer
+
+The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `foreman-installer --help`, or by running `foreman-installer -i` for interactive mode.
+More examples are given in the Installation Options section.
+The `-v` option disables the progress bar and displays all changes.
+
+.Procedure
+
+* Run the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario}
+----
+
+When the installer has completed, details are printed about where to find Foreman and the Smart Proxy.
```

Comment:
```suggestion
When the installer has completed, details are printed about where to find {ProjectServer} and {SmartProxy}.
```

I am also unsure about the wording "are printed".

Reply:
Agree, re: "are printed". Will check.

Reply:
I replaced the last sentence with the text that is used in the d/s installation guide.